.github/workflows/checks.yml

```diff
@@ -45,17 +45,11 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      - name: Workspace lint
-        run: npx nx workspace-lint
```

Comment:
This was part of the recommendations at some point from NX, I think it just makes sure that the project structure is coherent. Might be worth checking if it's still recommended or not.

Reply:
Ok I'll have a look, but if you run it now, it says that it is depecrated

```
 >  NX   "nx workspace-lint" has been deprecated.

   Nx has been reworked to make sure that the issues workspace-lint used to catch are not longer possible.
   Remove "nx workspace-lint" from your CI.
```